Library/Homebrew/test/os/mac/version_spec.rb

```diff
@@ -40,7 +40,7 @@
     expect(version).to be < Version.create("10.15")
   end
 
-  context "after Big Sur" do
+  context "when after Big Sur" do
```

Comment:
Another one that I'm not thrilled with, but is certainly fine to merge.

```
OS::Mac::Version when after Big Sur
```

Reply:
Yeah I know, some of them are a bit odd.